api/proto/teleport/legacy/types/types.proto

```diff
@@ -6197,7 +6197,7 @@ message GCPMatcher {
   repeated string Types = 1 [(gogoproto.jsontag) = "types,omitempty"];
   // Locations are GKE locations to search resources for.
   repeated string Locations = 2 [(gogoproto.jsontag) = "locations,omitempty"];
-  // Tags are GCP labels to match.
+  // Tags are an alias for Labels.
```

Comment:
I'm not sure what this means. Are you supposed to configure tags, labels, or both? What happens if they differ? Why do we need two fields to begin with?

Reply:
@zmb3 This is a result of poor naming choice on our part. GCP has a concept of both "tags" and "labels". Our auto-discovery looks at "labels" but the config field was called "tags" so it was very confusing. Now we're basically deprecating "tags" in our config and replacing with "labels" but we can't remove it completely for compatibility.

@atburke Let's make it clear that this field is obsolete, not supposed to be used by users and only exists for backwards compatibility.